model/src/main/java/org/jellyfin/apiclient/model/dlna/ConditionProcessor.java

```diff
@@ -125,9 +125,9 @@ private boolean IsConditionSatisfied(ProfileCondition condition, String currentV
 			case EqualsAny:
 				return ListHelper.ContainsIgnoreCase(expected.split("[|]", -1), currentValue);
 			case Equals:
-				return StringHelper.equalsIgnoreCase(currentValue, expected);
+				return currentValue.equalsIgnoreCase(expected);
```

Comment:
There is a check above to ensure `currentValue` is not null, so this should be fine.